.travis.yml

```diff
@@ -126,7 +127,6 @@ before_install:
 install:
     - mkdir -p ~/.config/matplotlib
     - echo "backend:agg" > ~/.config/matplotlib/matplotlibrc
-    - ./scripts/travis/install-openblas.sh $OPENBLAS_ROOT
```

Comment:
Maybe try building a different version of OpenBLAS?

A BLAS library is going to be required for all the frameworks. Since Travis doesn't have GPUs, we can't use cuBLAS, etc. You need ATLAS or OpenBLAS.